contributors/guide/contributor-cheatsheet/README-hi.md

```diff
@@ -0,0 +1,322 @@
+# कुबेरनेट्स योगदानकर्ता चीट शीट
+
+कुबेरनेट्स परियोजना में उपयोग किए जाने वाले कुबेरनेट्स,टिप्स, ट्रिक्स और सामान्य सर्वोत्तम प्रथाओं में योगदान करते समय सामान्य संसाधनों की एक सूची। यह एक "TL;DR" या आपके GitHub योगदान अनुभव को बेहतर बनाने के लिए उपयोगी जानकारी का त्वरित संदर्भ है।
+
+**विषय - सूची**
+
```

Comment:
_This is a general comment for TOC below_

As markdown convert heading into an anchor, so I guess, the links also need to be updated in Hindi language too.